scripts/feature_tests/upgrade/upgrade.sh

```diff
@@ -34,25 +34,21 @@ pushd "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade" || exit
 source 1cp_1w_bootDiskImage_cluster_upgrade.sh
 popd || exit
 
-# Deploy a fresh metal3-dev-env after each test case
-# to overcome environmental flakiness
-pushd "${METAL3_DEV_ENV_DIR}" || exit
-make clean
-make setup_env
+pushd "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade" || exit
```

Comment:
Would it be possible to source it form by specifying absolute path to the `clean_setup_env.sh`?

Reply:
can be ubuntu or centos so the relative path is more flexible

Reply:
My point was that we could do  `export "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade/clean_setup_env.sh"`. Os is there something that `clean_setup_env.sh` will need from /upgrade folder?

Reply:
Ok. Referring to https://github.com/metal3-io/metal3-dev-env/pull/448/files/80adc0d4f47056f987b17f5994d6a8f1f1e174d1#r485412338.
clean_setup_env.sh doesn't need anything from upgrade folder. The clean_setup_env.sh script should be moved to some common directory if it is seen to be beneficial for airship CI jobs.